staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/validation_test.go

```diff
@@ -77,6 +77,81 @@ func TestValidationExpressions(t *testing.T) {
 				"self.val7 == 1.0",
 			},
 		},
+		{name: "numeric comparisons",
+			obj:    objs(int64(5), int64(10), int64(15), float64(10.0), float64(10.0), float64(10.0), int64(10), int64(10), int64(10)),
+			schema: schemas(integerType, integerType, integerType, numberType, floatType, doubleType, numberType, floatType, doubleType),
```

Comment:
might be helpful to stack these with comments about which identifier they are and what scenario they represent

```go
obj: objs(
  int64(5), // val1, integer type, integer value
  int64(10), // val2, integer type, integer value
  int64(15), // val3, integer type, integer value
  float64(10.0), // val4, number type, parsed from decimal literal
  float64(10.0), // val5, float type, parsed from decimal literal
  float64(10.0), // val6, double type, parsed from decimal literal
  int64(10), // val7, number type, parsed from integer literal
  int64(10), // val8, float type, parsed from integer literal
  int64(10), // val9, double type, parsed from integer literal
),

Reply:
Thx. I've applied this.